frontend/src/pages/admin/Users/General.vue

```diff
@@ -56,8 +56,10 @@ export default {
             nextCursor: null,
             columns: [
                 { label: 'User', class: ['flex-grow'], key: 'name', component: { is: markRaw(UserCell) }, sortable: true },
-                { label: 'Password Expired', class: ['w-56', 'text-center'], key: 'password_expired', sortable: true },
-                { label: 'Email Verified', class: ['w-56', 'text-center'], key: 'email_verified', sortable: true },
+                { label: 'Password Expired', class: ['w-32', 'text-center'], key: 'password_expired', sortable: true },
+                { label: 'Email Verified', class: ['w-32', 'text-center'], key: 'email_verified', sortable: true },
+                { label: 'SSO', class: ['w-32', 'text-center'], key: 'sso_enabled', sortable: true },
```

Comment:
Where is `sso_enabled` coming from?

Reply:
It's a pre-existing flag on the user object. Thought it useful to add to what we expose here... although it's getting to a point where this table needs a bit of redesign.